UPDATING.md

```diff
@@ -24,6 +24,12 @@ assists users migrating to a new version.
 
 ## Airflow Master
 
+### Deprecate legacy UI in favor of FAB RBAC UI
+Previously we were using two versions of UI, which were hard to maintain as we need to implement/update the same feature
+in both versions. With this change we've removed the older UI in favor of Flask App Builder RBAC UI. No need to set the 
+RBAC UI explicitly in the configuration now as this is the only default UI. 
```

Comment:
Probably worth calling out explicitly that custom auth backends will need re-writing to target FAB

Reply:
good point, updated.